language/move-prover/docgen/src/docgen.rs

```diff
@@ -1081,7 +1082,12 @@ impl<'env> Docgen<'env> {
     /// Generates documentation for a function signature.
     fn function_header_display(&self, func_env: &FunctionEnv<'_>) -> String {
         let name = self.name_string(func_env.get_name());
-        let visibility = if func_env.is_public() { "public " } else { "" };
+        let visibility = match func_env.visibility() {
+            FunctionVisibility::Public => "public ",
```

Comment:
Perhaps we should just implement Display for FunctionVisibility in the canonical way as here, then we can remove some code. That we use "pub" instead of "public" in the bytecode crate should not block us here, you can just regenerate all the baseline tests in this crate with `UPBL=1 cargo test`.

Reply:
Had a look, and I'll implement it as a method in the Move Model. The returned strings are a bit odd for a `std::fmt::Display` due to an empty string being returned for private functions, and the ending space.